component/site/views/checkout/tmpl/default_address.php

```diff
@@ -37,9 +39,13 @@
                     if ($billingAddresses) {
                         ?>
                         <a class="modal btn btn-primary" href="<?php echo $editbill; ?>"
-                           rel="{handler: 'iframe', size: {x: 720, y: 470}}"><?php echo JText::_(
-                                'COM_REDSHOP_EDIT'
-                            ); ?></a>
+                           rel="{handler: 'iframe', size: {x: 720, y: 470}}"><?php echo JText::_('COM_REDSHOP_EDIT'); ?>
+                        </a>
+	                    <?php if ($auth['users_info_id'] && Redshop::getConfig()->getInt('ENABLE_CLEAR_USER_INFO') == 1): ?>
+                            <button type="button" class="btn btn-primary" name="clear_user_info" onclick="javascript:clearUserInfo();">
+			                    <?php echo JText::_('COM_REDSHOP_CLEAR_USER_INFO'); ?>
+                            </button>
+	                    <?php endif;?>
```

Comment:
@hoang-tran-redweb : `COM_REDSHOP_CLEAR_USER_INFO` not defined language yet.
![image](https://user-images.githubusercontent.com/6948022/91158529-f67abd80-e6f0-11ea-8a3b-42f3211b1782.png)

Reply:
@anhredweb I fixed. please review again.